agreements/create_lots.sql

```diff
@@ -0,0 +1,201 @@
+/*
+Title       create_lots.sql
+Date        June 15th 2020
+Author      Trevor Cummings
+Description This file is a script to create test data for lots         
+*/
+
+--RM3733 CA FK 1, Has 6 lots
+INSERT INTO lOTS
+(lot_id,
+ lot_number,
+ lot_name,
+ lot_description,          
+ commercial_agreement_id, -- FK back to commercial agreements
+ lot_type,                -- Valid values Services, Products or Products and Services
+ start_date,
+ end_date)
+values 
+(1,
+ 'Lot 1',
+ 'Technology Hardware',
+ 'Technology Hardware description',
+ 1,
+ 'Products',
```

Comment:
According to Dave's API spec - Lot type is singular - i.e. 'product', 'service' or 'product and service'. Having them the same make mapping easier if possible 
![image](https://user-images.githubusercontent.com/2317320/84717584-262d7c80-af6e-11ea-9659-930f739ab459.png)